txscript/engine.go

```diff
@@ -1060,6 +1100,31 @@ func (vm *Engine) Execute() (err error) {
 
 			return dstr + astr
 		}))
+
+		if vm.StepCallback != nil {
+			scriptIdx := vm.scriptIdx
+			opcodeIdx := vm.opcodeIdx
+
+			// In case the execution has completed, we keep the
```

Comment:
May be good to have this documented in `StepInfo` as well.

Reply:
Good call: 3033ba39